### PR TITLE
DOCUMENTATION: Readme Cleanup Add Nuget Refs

### LINKED ISSUE #133: DOCUMENTATIONS: Refer to Nuget Package in Readme.md

### PR DESCRIPTION
**Pull request by glhays**
Closes #133

@hassanhabib @ElbekDeveloper 
Took some time today and reviewed Nuget packages that are using images. All seem to be using logo graphic some with an incorporated package name and others not. 
In conclusion Elbek is correct markdown syntax does not support centering, and all that found in nuget were left aligned. This PR incorporates the logo in markdown, nuget package links, and the latest videos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
 # Standard.AI.OpenAI
 
-<div align=center>
-	<img width="300" src="https://raw.githubusercontent.com/hassanhabib/OpenAI.NET/main/Standard.AI.OpenAI/artificial-intelligence.png" />
-</div>
+![](https://raw.githubusercontent.com/hassanhabib/OpenAI.NET/main/Standard.AI.OpenAI/artificial-intelligence.png)
```
Comment (mabroukmahdhi) [inline README.md:3]: This code with display the logo with a large size and it will not be in the center.

Comment (hassanhabib) [reply README.md:3]: I agree with @mabroukmahdhi 

I saw others do this:
https://github.com/NuGetPackageExplorer/NuGetPackageExplorer

Comment (hassanhabib) [reply README.md:3]: This piece is also a separate issue. You should revert this and keep the other code and then we can merge it.

Comment (glhays) [reply README.md:3]: @mabroukmahdhi Thank you for the catch.
@hassanhabib Thank you, will do.

File: README.md [hunk 2]
```diff
@@ -4,9 +4,10 @@
 	<img width="300" src="https://raw.githubusercontent.com/hassanhabib/OpenAI.NET/main/Standard.AI.OpenAI/artificial-intelligence.png" />
 </div>
 
-<br />
+</br>
```
Comment (hassanhabib) [inline README.md:7]: I think it's `<br />` look at MDN docs

image

Comment (glhays) [reply README.md:7]: My bad, typo dahhh! Had html on the brain.

**Comment by hassanhabib** [comment]
@glhays I see some tags for nuget that shows even the amount of downloads on a given nuget package.
Can you find out how to do that?

**Comment by glhays** [comment]
Sure no problem.

Also can you take a look at my comment Elbek's issue #138 I know that Christo said the <p> tag is not allowed in markdown which is true and false. As long as it is separated from markdown syntax with blank lines it is, just not embedded within the markdown.
My honest opinion is and I remember other parsing engines not liking div tags and they just spit them out as text and everything between them.
I am when a free moment arises going to mock a library up and do some test publishing to nuget and see what may work.

File: README.md (hunk 2 above)
Comment (glhays) [reply README.md]: Done

**Comment by glhays** [comment]
@hassanhabib downloads badge added.

**Pull request merged**